lib/redis/cluster.rb

```diff
@@ -137,6 +137,7 @@ def send_command(command, &block)
       when 'wait'     then @node.call_master(command, &block).reduce(:+)
       when 'keys'     then @node.call_slave(command, &block).flatten.sort
       when 'dbsize'   then @node.call_slave(command, &block).reduce(:+)
+      when 'scan'     then _scan(command, &block)
```

Comment:
@byroot what about zscan/hscan/sscan, will it work? may I should add more tests? I'm not very familiar with internals.

Reply:
More tests wouldn't hurt.